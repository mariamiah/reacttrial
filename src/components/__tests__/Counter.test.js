import React from 'react';
import Counter from '../counter';
import ReactDOM from 'react-dom';
import { configure, shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({adapter: new Adapter()});

it('renders without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Counter />, div)
})

test('renders a header', ()=>{
    const wrapper = shallow(<h1>My Counter</h1>);
    expect(wrapper).toMatchSnapshot();
})
// const wrapper = shallow(<Counter />)
// // it('renders without crushing', ()=>{
// //     const div = document.createElement('div');
// //     ReactDOM.render(<Counter />, div)
// // })