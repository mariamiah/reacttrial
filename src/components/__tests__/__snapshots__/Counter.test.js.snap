// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a header 1`] = `
<h1>
  My Counter
</h1>
`;
